README.md

```diff
@@ -225,57 +201,49 @@ If you want the audit cookbook to directly report to Chef Compliance, set the `r
  * `insecure` - a `true` value will skip the SSL certificate verification when retrieving access token. Default value is `false`
 
 ```ruby
-"audit": {
-  "reporter": "chef-compliance",
-  "server": "https://compliance-fqdn/api",
-  "owner": "my-comp-org",
-  "refresh_token": "5/4T...g==",
-  "insecure": false,
-  "profiles": [
-    {
-      "name": "windows",
-      "compliance": "base/windows"
-    }
-  ]
-}
+default['audit']['reporter'] = 'chef-compliance'
+default['audit']['server'] = 'https://compliance-fqdn/api'
+default['audit']['owner'] = 'my-comp-org'
+default['audit']['refresh_token'] = '5/4T...g=='
+default['audit']['profiles'].push(
+  {
+    'name': 'windows',
+    'compliance': 'base/windows',
+  }
+)
 ```
 
-Instead of a refresh token, it is also possible to use a `token` that expires in 12h after creation .
+Instead of a refresh token, it is also possible to use a `token` that expires in 12h after creation.
```

Comment:
Instead of a refresh token, you can also use a `token` that expires 12 hours after its creation.

Otherwise, we are discussing biblical time :-)

Reply:
Got a good chuckle out of this. Compliance is EOL though, as of 1,000,000 BC.